### PR TITLE
Goshko HW9

### PR DESCRIPTION
**Pull request by Ant3000**

### CODE REVIEW

File: GOshko/Ghw9/GoshkoHW9Task1.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+def imp_mod():  # task 1.1
+    import calculator
```
Comment (katemalash) [inline GOshko/Ghw9/GoshkoHW9Task1.py:2]: it's good practice to import modules outside the functions

File: GOshko/Ghw9/GoshkoHW9Task1.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+from calculator import *  # task 1.3
+
+imp_mod()  # task 1.1
+print()
+imp_2()  # task 1.2
+print()  # task 1.3
+print(sum(20, 10))
+print(dif(20, 10))
+print(product(20, 10))
+print(fraction(20, 10))
```
Comment (katemalash) [inline GOshko/Ghw9/GoshkoHW9Task1.py:24]: Good way

File: GOshko/Ghw9/GoshkoHW9Task4.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+@sandwich
+def cheese():
+    print('bread, butter, cheese')
+
+
+@sandwich
+def tomato():
+    print('bread, butter, tomato')
+
```
Comment (katemalash) [inline GOshko/Ghw9/GoshkoHW9Task4.py:32]: Nice sandwiches

**Review by katemalash — APPROVED** [review]
Good Job. Now you can merge this pull-request

File: GOshko/Ghw9/GoshkoHW9Task1.py (hunk 1 above)
Comment (Ant3000) [reply GOshko/Ghw9/GoshkoHW9Task1.py:2]: and how else to make 3 tasks on import in one file.py ???

**Pull request merged**